patches/chrome-android-java-AndroidManifest.xml.patch

```diff
@@ -90,7 +90,23 @@ index 364f021a07e1dd20463ccadeacecbfd1361cb93c..8477af119562f001a974b7e08180c849
  
          <!-- Bookmarks widget -->
          <receiver android:name="com.google.android.apps.chrome.appwidget.bookmarks.BookmarkThumbnailWidgetProvider"
-@@ -1202,6 +1218,7 @@ by a child template that "extends" this file.
+@@ -932,6 +948,7 @@ by a child template that "extends" this file.
+             android:exported="false" />
+ 
+         <!-- Search widget -->
++        <!-- 
```

Comment:
What would be the reason for this removal? I have looked into the PR description, commit message, and it doesn't seem to mention. The linked issue also doesn't mention it.

Reply:
We have decided to remove `Search widget` as we have worked on Brave search widget which provides more details than what we inherit from chromium. 
We would be following up with Brave specific widgets.
CC @timchilds

Reply:
Please then consider clarifying that in the PR description.

Reply:
i have updated the PR with more details.